safety-architecture/tools/callgraph-tool/AUTHORS

```diff
@@ -0,0 +1,10 @@
+<!--
+SPDX-FileCopyrightText: 2020 callgraph-tool authors. All rights reserved
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# This file contains copyright holder of the callgraph-tool repository
```

Comment:
Should this be changed: 
"contains copyright holder" ==> "contains copyright holders"
already anticipating that there might be more than one copyright holder?